src/screens/Shared/ViewBranchMachine/components/TabPaymentsReceived/index.tsx

```diff
@@ -0,0 +1,246 @@
+import { Summary } from './components/Summary';
+
+const columns: ColumnsType = [
+	{ title: 'Datetime', dataIndex: 'datetime' },
+	{ title: 'Receipt Type', dataIndex: 'receipt_type' },
+	{ title: 'Reference Number', dataIndex: 'invoice' },
+	{ title: 'Payment', dataIndex: 'payment' },
+	{ title: 'Cashier', dataIndex: 'cashier' },
+	{ title: 'Mode of Payment', dataIndex: 'mode_of_payment' },
+	{ title: 'Remarks', dataIndex: 'remarks' },
+];
+
+const invoiceTypes = {
+	sales_invoice: 'Sales Invoice',
+	charge_invoice: 'Charge Invoice',
+};
+
+const modeOfPayment = {
+	cash: 'Cash',
+	credit_pay: 'Credit Card',
+	check: 'Check',
+	others: 'Others',
+};
+
+const voidedStatuses = [
+	transactionStatuses.VOID_CANCELLED,
+	transactionStatuses.VOID_EDITED,
+];
+
+interface Props {
+	branchMachineId: number;
+}
+
+export const TabPaymentsReceived = ({ branchMachineId }: Props) => {
+	// STATES
+	const [dataSource, setDataSource] = useState([]);
+	const [selectedTransaction, setSelectedTransaction] = useState(null);
+	const [
+		selectedCollectionReceipt,
+		setSelectedCollectionReceipt,
+	] = useState<CollectionReceipt | null>(null);
+
+	// CUSTOM HOOKS
+	const { params, setQueryParams } = useQueryParams();
+	const { data: siteSettings } = useSiteSettingsNew();
+	const {
+		data: { transactions, total },
+		error: transactionsError,
+		isFetching: isFetchingTransactions,
+		isFetchedAfterMount: isTransactionsFetchedAfterMount,
+	} = useTransactions({
+		params: {
+			...params,
+			branchMachineId,
+			timeRange: params?.timeRange || timeRangeTypes.DAILY,
+		},
+		options: refetchOptions,
+	});
+
+	const {
+		data: collectionReceiptsData,
+		isFetching: isFetchingCollectionReceipts,
+		error: collectionReceiptsError,
+	} = useCollectionReceipts({
+		params: {
+			...params,
+			branchMachineId,
+			timeRange: (params?.timeRange || timeRangeTypes.DAILY) as string,
+		},
+		options: refetchOptions,
+		serviceOptions: { baseURL: getLocalApiUrl() },
+	});
+
+	console.log(collectionReceiptsData);
+
+	const receiptsArray = collectionReceiptsData?.list || [];
+
+	const receiptsMap = {};
+	receiptsArray.forEach((receipt) => {
+		if (receipt.order_of_payment) {
+			receiptsMap[receipt.order_of_payment.id] = receipt;
+		}
+	});
+
+	// METHODS
+	useEffect(() => {
+		if (transactions && collectionReceiptsData) {
+			// Convert transactions to table rows
+			const transactionsData = transactions.map((transaction) => ({
+				key: `transaction-${transaction.id}`,
+				datetime: formatDateTime(transaction.datetime_created),
+				receipt_type: 'Cash Sales Invoice',
+				invoice: (
+					<Button
+						className="pa-0"
+						type="link"
+						onClick={() => setSelectedTransaction(transaction)}
+					>
+						{transaction.invoice?.or_number}
+					</Button>
+				),
+				payment: formatInPeso(transaction.payment?.amount_tendered),
+				cashier: getFullName(transaction.teller),
+				mode_of_payment: modeOfPayment[transaction.payment?.mode],
+				remarks: 'N/A',
```

Comment:
Don't use magic strings/numbers. Store this in a variable (preferrably stored in constants.tsx in `ejjy-global`) and add it here.